app/build.gradle

```diff
@@ -191,6 +192,7 @@ android {
     defaultConfig {
         minSdkVersion 9
         targetSdkVersion 25
+        multiDexEnabled true
```

Comment:
Yea, the app won't run without it. It seems like adding the jsoup library just took us over the limit. We could try to go through and see if there are any libraries we can remove, but unless we removed a lot, we'd probably just hit this again sometime soon.